WDAC-Policy-Wizard/app/src/Helper.cs

```diff
@@ -1301,6 +1312,30 @@ public void AddNewSeparationLine(string subTitle)
             }
         }
 
+        /// <summary>
+        /// Removes any path of the form :\Users\john.doe\xyz with %USERPROFILE%
+        /// </summary>
+        /// <param name="msg">Sanitized message</param>
+        /// <returns></returns>
+        private string SanitizeMsg(string msg)
```

Comment:
This functional fails to sanitize something like ":\Users\john.doe\Users\john.doe\Users\john.doe"